src/lib.rs

```diff
@@ -1,3 +1,118 @@
+//!
+//! # Momento
+//! Welcome to the Momento Rust SDK!
+//!
+//! This crate contains a `CacheClient` for interacting with a serverless Momento Cache, and a
+//! `TopicClient` for interacting with serverless Momento Topics (pub/sub messaging). For more detailed
+//! information see the [Momento documentation](https://docs.momentohq.com).
+//!
+//! Construct instances of these clients using `CacheClient::builder()` and `TopicClient::builder()`.
```

Comment:
Might be nice to turn these into links to the builders

```suggestion
//! Construct instances of these clients using [CacheClient::builder()] and [TopicClient::builder()].
```

Reply:
Thank you! I didn't know that syntax.